config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -348,6 +348,43 @@ periodics:
           cpu: "4"
       securityContext:
         privileged: true
+- annotations:
+    fork-per-release-generic-suffix: "true"
+    testgrid-dashboards: google-windows, sig-release-1.16-informing, sig-windows
+    testgrid-tab-name: gce-windows-beta
+    description: Runs tests on a Kubernetes 1.16 cluster with Windows nodes on GCE
+  decorate: true
+  extra_refs:
+  - org: yujuhong
```

Comment:
It looks like everything else in these files refer to kubernetes/kubernetes except for the Windows job. I think long term, the stuff in https://github.com/yujuhong/gce-k8s-windows-testing needs to be integrated with k/k so we can remove the dependency on the Yuju's personal repos.

Reply:
Yep. We should move it somewhere else. I think we discussed this briefly with @PatrickLang a long time ago, but it was decided that the vendor-specific configs should not be hosted in the windows-testing repo. Maybe we can re-visit that decision?

Reply:
do you think that blocks this PR? I don't recall the specifics but I'd be open to consolidating the extra things needed for test jobs in one place.

Reply:
Not a blocking issue. Tracking this with the other stuff for test jobs SGTM.